packages/frontend/src/static/_redirects

```diff
@@ -4,4 +4,5 @@
 /project/*                  /projects/:splat
 /projects/leverj            /projects/gluon
 /projects/fuel              /projects/fuelv1
-/projects/zkswapv2          /projects/zkspace
\ No newline at end of file
+/projects/zkswapv2          /projects/zkspace
+/api/projects/:project/tvl/assets/:asset          https://api.l2beat.com/api/projects/:project/tvl/assets/:asset
```

Comment:
What about serving locally?

Reply:
I guess we could have a `API_BASE_URL` env var, which would prepended (if set in the environment) to paths in the frontend that should essentially point to api e.g. here https://github.com/l2beat/l2beat/pull/548/files#diff-e19d3950687c41524d803f6f4da536d6ff3a3604efb7c54a168004cca148502aR19